src/openzaak/components/documenten/admin.py

```diff
@@ -40,6 +46,9 @@ def get_not_lock_display(self, obj) -> bool:
     get_not_lock_display.short_description = "free to change"
     get_not_lock_display.boolean = True
 
+    def get_viewset(self, request):
```

Comment:
Why is this here?

Reply:
`EnkelvoudigInformatieObjectCanonical` doesn't have matching viewset, so the changes in the admin can't be logged. But in the admin page it has inline objects - `EnkelvoudigInformatieObject` and `Gebruiksrechten` which have related viewsets with AuditTrail. So we can log only changes in the inline objects here (it's a special case).

If we don't override `get_viewset` method the exception will be raised since this class doesn't have `viewset` attribute.